internal/mq/mqimpl/rocksmq/client/producer_impl.go

```diff
@@ -50,11 +51,23 @@ func (p *producer) Topic() string {
 }
 
 // Send produce message in rocksmq
-func (p *producer) Send(message *ProducerMessage) (UniqueID, error) {
+func (p *producer) Send(message *mqwrapper.ProducerMessage) (UniqueID, error) {
+	// NOTICE: this is the hack.
+	// we should not unmarshal the payload here but we can not extend the payload byte
+	payload := message.Payload
+	header, err := UnmarshalPayload(message.Payload)
+	if err == nil && header != nil && header.Base != nil {
```

Comment:
> output log and don't send it if the error is not nil

this will not happen online.
the err catch is only for some test where the payload is not a protobuf or messsage